strr-web/pages/create-account.vue

```diff
@@ -163,6 +163,28 @@ const ownershipToApiType = (type: string | undefined): string => {
   return ''
 }
 
+const rentalUnitSpaceTypeToApiType = (type: string | undefined): RentalUnitSpaceTypeE | '' => {
```

Comment:
Not sure if this is needed. What is this trying to accomplish? Match String back to Type?

Reply:
i can pass them similar to 
formState.propertyDetails.isUnitOnPrincipalResidenceProperty